src/test/java/ai/rev/speechtotext/integration/SubmitJobTest.java

```diff
@@ -122,14 +122,14 @@ public void SubmitJobUrl_UrlAndOptionsAreSpecified_ReturnsRevAiJobInProgress()
       throws IOException {
     RevAiJobOptions revAiJobOptions = getJobOptions();
 
-    RevAiJob revAiJob = apiClient.submitJobUrl(MEDIA_URL, revAiJobOptions);
+    RevAiJob revAiJob = apiClient.submitJobUrl(revAiJobOptions);
 
     assertRevAiJob(revAiJob);
   }
 
   @Test
   public void SubmitJobUrl_OnlyUrlIsSpecified_ReturnsRevAiJobInProgress() throws IOException {
-    RevAiJob revAiJob = apiClient.submitJobUrl(MEDIA_URL, null);
+    RevAiJob revAiJob = apiClient.submitJobUrl(SOURCE_URL);
```

Comment:
As long as we support the old parameters, even if they are deprecated, I think we should add new tests instead of modifying the existing tests.

Alternatively, you can leave as you have here and add one more tests to check that the old parameter still works